homeassistant/components/elmax/common.py

```diff
@@ -105,9 +106,7 @@ async def _async_update_data(self):
                 # If the panel is online, proceed with fetching its state
                 # and return it right away
                 if panel.online:
-                    status = await self._client.get_panel_status(
-                        control_panel_id=panel.hash, pin=self._panel_pin
-                    )  # type: PanelStatus
+                    status = await self._retriable_update(panel)
```

Comment:
Would be better to implement this in the package instead

Reply:
Done. I've opened a new PR for this.
https://github.com/home-assistant/core/pull/91926